manifest.json

```diff
@@ -52,5 +52,6 @@
         "description": "Enter with the ID (GTM-XXXX) from your Google Tag Manager container if you want to automatically use it"
       }
     }
-  }
-}
+  },
+  "$schema": "https://raw.githubusercontent.com/vtex/node-vtex-api/master/gen/manifest.schema"
```

Comment:
what does this mean?

Reply:
It's from toolbelt it adds this automatically so we can have autocomplete while typing the manifest.